BricksAndHearts.UnitTests/ServiceTests/Api/ApiServiceTests.cs

```diff
@@ -14,23 +15,20 @@
 
 namespace BricksAndHearts.UnitTests.ServiceTests.Api;
 
-public class ApiTests
+public class ApiServiceTests
 {
     // A fake options
     private static readonly IOptions<AzureMapsOptions> _options = A.Fake<IOptions<AzureMapsOptions>>();
+    private static readonly ILogger<AzureMapsAzureMapsApiService> _logger = A.Fake<ILogger<AzureMapsAzureMapsApiService>>();
+    private static readonly HttpMessageHandler _messageHandler = A.Fake<HttpMessageHandler>();
+    private static readonly HttpClient _httpClient = new HttpClient(_messageHandler);
+    
     // A real API service
-    private readonly AzureMapsAzureMapsApiService _underTest = new(null!, _options);
+    private static readonly AzureMapsAzureMapsApiService _underTest = new(_logger, _options,_httpClient);
```

Comment:
Avoid static test fakes/services - they will be reused between tests which can lead to weird bugs. Make them in the constructor instead